CHANGELOG.md

```diff
@@ -25,4 +25,8 @@ Released on ????? ?th, 20??.
 
 ### Added
 
-- `DockerSettings` task - [#3](https://github.com/PrefectHQ/prefect-docker/pull/3)
+- `DockerSettings` block - [#3](https://github.com/PrefectHQ/prefect-docker/pull/3)
+
+### Changed
+
+- `DockerSettings` renamed to `DockerHost` - [#13](https://github.com/PrefectHQ/prefect-docker/pull/13)
```

Comment:
Just a comment, doesn't need to change in this PR: when we release this collection, I think we should consolidate these two entries into one entry under just Added.

Reply:
Which PR number should I reference?

Reply:
I think we can reference both